src/app/components/landing/landing.component.ts

```diff
@@ -1,11 +1,44 @@
+import { NgClass } from '@angular/common';
+import { ReturnStatement } from '@angular/compiler';
 import { Component, OnInit } from '@angular/core';
+import { CapacityService } from 'src/app/services/capacity.service';
 
 @Component({
   selector: 'app-landing',
   templateUrl: './landing.component.html',
   styleUrls: ['./landing.component.scss']
 })
-export class LandingComponent {
-  constructor() {
+export class LandingComponent implements OnInit {
+  constructor(private capacityService: CapacityService) {
   }
-}
+  restaurantName: string = '';
+  welcomeDescription: string = '';
+  beginMessage: string = '';
+  currentCapacity: number = 0;
+  capacityMessage: string = '';
+  orderIsDisabled: boolean = false
+
+  ngOnInit(): any {
+    this.currentCapacity = this.capacityService.getCapacity();
+    this.capacityMessage = this.oneOrZeroSeat();
+    this.orderIsDisabled = this.disabledButton();
+    this.restaurantName = 'The Food Shop';
+    this.welcomeDescription = 'Welcome to ThisFoodApp, the easiest and fastest way to order your favorite food. With ThisFoodApp, you can browse menus, customize your orders, pay securely, and get your food in minutes';
+    this.beginMessage = "Let's begin!";
+  }
+
+  oneOrZeroSeat(): string {
```

Comment:
This method name could be better. Something like 'getCapacityMessage' or 'createCapacityMessage' would describe better what the method actually does.